### PR TITLE
Added custom options.

### PR DESCRIPTION
**Pull request by dima-bzz**
Signed-off-by: Dmitry Mazurov <dimabzz@gmail.com>

## Description

Added custom options.

## Motivation and context

This is a new feature that adds the ability to specify arbitrary options.

## How has this been tested?

Ubuntu 18.04
Php 7.2.24
Laravel 6.11.0

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [ ] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by slavarazum** [comment]
In which cases do you see this as useful? Actions chain depend on the current environment, when we need to have a several options for the same environment?

**Comment by dima-bzz** [comment]
@slavarazum Suppose I need to perform an update in the development environment and recreate the database, it will be convenient to specify additional parameters for this. But I do not always need to reinstall the database. This is an example. And so it adds flexibility to packages, so it allows you to perform different actions with different options.

File: src/Actions/External.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function run(): bool
             }
         } : null);
 
-        $error = $Process->getErrorOutput();
+        $error = empty($Process->getErrorOutput()) ? $Process->getOutput() : $Process->getErrorOutput();
```
Comment (slavarazum) [inline src/Actions/External.php]: Why this is appeared in this PR?

Comment (dima-bzz) [reply src/Actions/External.php]: This was an erroneous merge. Should I delete it or make changes?

**Pull request closed without merging**